docker-compose.yml

```diff
@@ -1,18 +1,23 @@
-version: '2'
+version: '2.1'
```

Comment:
For this to work, we need at least Docker Engine `1.12.0` - once we fix the Docker Compose issue we can output the version of the docker engine to verify.